integreat_compass/cms/models/interactions/comment.py

```diff
@@ -12,7 +12,7 @@ class Comment(AbstractBaseModel):
     Data model representing a Comment.
     """
 
-    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    creator = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
Does this really make what we want? If a user is deleted, I think his/her comments should stay. :thinking:

Reply:
Two options what we could do here:
- `SET_NULL`: would make the most sense, in our templates we could simply show something like `[deleted user]` as their name if the reference is null. However, this requires creator to be nullable which I don't really like...
- `SET()`: we could also set a "fallback" user here... Avoids the nullable field but feels hacky.

Reply:
In my opinion, the first option with a nullable field and `SET_NULL` makes more sense :+1:

Reply:
Alternatively, you could also keep old users for reference, and only disable their login by setting `is_active=False`...
Then, you could use `PROTECT` instead of `CASCADE` or `SETNULL`.

Reply:
From a techincal perspective yes, but since the users do have a right to be forgotten, we'd have to at least also remove their email and display name, at which point, deleting them properly might make more sense? 😅